diff --git a/web/app.js b/web/app.js
index b04eebff..91d5241b 100644
--- a/web/app.js
+++ b/web/app.js
@@ -1423,7 +1423,7 @@ let PDFViewerApplication = {
 let validateFileURL;
 if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
   const HOSTED_VIEWER_ORIGINS = ['null',
-    'http://mozilla.github.io', 'https://mozilla.github.io'];
+    'http://mozilla.github.io', 'https://mozilla.github.io', 'file://', 'qrc://'];
   validateFileURL = function validateFileURL(file) {
     if (file === undefined) {
       return;
diff --git a/web/app_options.js b/web/app_options.js
index 08fb2884..4a32b237 100644
--- a/web/app_options.js
+++ b/web/app_options.js
@@ -35,7 +35,7 @@ const defaultOptions = {
   },
   defaultUrl: {
     /** @type {string} */
-    value: 'compressed.tracemonkey-pldi-09.pdf',
+    value: '',
     kind: OptionKind.VIEWER,
   },
   defaultZoomValue: {
